frontend/pages/hosts/details/cards/Software/InstallStatusCell/InstallStatusCell.tsx

```diff
@@ -59,12 +61,18 @@ export const INSTALL_STATUS_DISPLAY_OPTIONS: Record<
   avaiableForInstall: {
     iconName: "install",
     displayText: "Available for install",
-    tooltip: ({ softwareName }) => (
-      <>
-        {softwareName ? <b>{softwareName}</b> : "Software"} can be installed on
-        the host. Select <b>Actions {">"} Install</b> to install.
-      </>
-    ),
+    tooltip: ({ softwareName, isAppStoreApp }) =>
+      isAppStoreApp ? (
+        <>
+          App Store app can be installed on the host. Select{" "}
+          <b>Actions &gt; Install</b> to install.
```

Comment:
Nit: Maybe change `&gt;` to `{">"}` here?

Reply:
changed